test/FastTests/Issues/RavenDB-6250.cs

```diff
@@ -72,7 +72,8 @@ public void All_performance_hints_has_details_providers()
             var operationWithoutDetails = new HashSet<PerformanceHintType>
             {
                 PerformanceHintType.None,
-                PerformanceHintType.Replication
+                PerformanceHintType.Replication,
+                PerformanceHintType.SlowIO
```

Comment:
Was this actually handled?

Reply:
![image](https://user-images.githubusercontent.com/1250431/35683282-c24dc986-0763-11e8-833b-24213f777fd9.png)

It is operation w/o details. This test is a trap to check if we have explicitily put performance hint to operationWithoutDetails  hash set or to alreadyHandledInStudio.

Reply:
Please ignore numbers in notification I hard coded invocation just to get this warning.